core/helpers/EEH_Sideloader.helper.php

```diff
@@ -84,7 +77,7 @@ private function _init($init)
         $props = array_merge($defaults, $init);
 
         foreach ($props as $key => $val) {
-            if (EEH_Class_Tools::has_property($this, $key)) {
+            if (property_exists($this, $key)) {
                 $setter = 'set' . $key;
                 $this->$setter($val);
```

Comment:
ohhhh... actually another possibly better way to do this would be to verify that the setter exists instead of checking for the property, ie:

```suggestion
            $setter = 'set' . $key;
            if (method_exists($this, $setter)) {
                $this->$setter($val);
```

since, although things would work absolutely fine the way you have things now, there's always the chance that someone could add a new property in the future that does not follow the correct setter naming schema (or not have a setter at all) and then try to pass that prop name within the `$init` array. So the `property_exists()` check would pass, but the subsequent attempt to call the setter would fail. So really we only care if the setter exists. This of course could then benefit from the addition of an error message to indicate that no setter exists for the provided property.

Sorry for bouncing this back again.